auth/contract/contract_test.go

```diff
@@ -222,4 +223,12 @@ func TestParseContractString(t *testing.T) {
 		assert.NotNil(t, err)
 		assert.True(t, errors.Is(err, ErrInvalidContractText))
 	})
+	t.Run("ok", func(t *testing.T) {
+		rawText := "NL:BehandelaarLogin:v1 Ondergetekende geeft toestemming aan Demo EHR om namens Toon & Zoonen en ondergetekende het Nuts netwerk te bevragen. Deze toestemming is geldig van dinsdag, 1 oktober 2019 13:30:42 tot dinsdag, 1 oktober 2019 14:30:42."
+		signedContract, err := ParseContractString(rawText, StandardContractTemplates)
+
+		require.NoError(t, err)
+		require.NotNil(t, signedContract)
+		assert.Equal(t, "", signedContract.Params[""])
```

Comment:
why checking the empty string? Would make sense to check if the params contain the actual "Toon & Zoon" information in the `legal_entity`.